Modules/Core/Common/src/itkPoolMultiThreader.cxx

```diff
@@ -86,12 +86,10 @@ PoolMultiThreader::PoolMultiThreader()
   }
 
   ThreadIdType defaultThreads = std::max(1u, GetGlobalDefaultNumberOfThreads());
-#if !defined(ITKV4_COMPATIBILITY)
   if (defaultThreads > 1) // one work unit for only one thread
```

Comment:
This is not the number of threads any, it's the number of work units only?

Reply:
I agree, but this is outside the scope of efforts in this PR.